pkg/skaffold/initializer/init.go

```diff
@@ -63,6 +63,7 @@ type Config struct {
 	EnableBuildpacksInit   bool
 	BuildpacksBuilder      string
 	Opts                   config.SkaffoldOptions
+	EnableNewInitFormat    bool
```

Comment:
> > pkg/skaffold/initializer/init.go:54:13: struct of size 400 bytes could be of size 392 bytes (maligned)
type Config struct {

This linter is pedantic! I love it! You can save 4 bytes by moving this up with the other `bool`s.

Reply:
oh yeah, right, this always gets me